utils/jwt.ts

```diff
@@ -18,10 +18,27 @@ export function validateJwt(req: Request, res: Response, next: NextFunction) {
   jwt.verify(token, process.env.TOKEN_SECRET as string, (err: VerifyErrors, decoded: any) => {
     if (err) {
       logger.error(err);
-      return res.sendStatus(403).end();
+      return res.status(403).json("refresh token is expired").end();
     }
 
     req.body.id = decoded.id;
     next();
   });
 }
+
+export function validateLobbyJwt(req: Request, res: Response, next: NextFunction) {
+  const authHeader = req.body.lobbyToken;
+  const token = authHeader && authHeader.split(' ')[1];
```

Comment:
i dont know if we need to split the auth header.. we can just make the input a token!

So body looks like 

```json
{
  "lobbyToken": TOKEN
}
```

Reply:
Ok sounds good